course_1_1/Program.cs

```diff
@@ -10,19 +10,331 @@ class Program
-            
+            //
+            //HoursInDay();
+            //
+            //
+            //ABetweenC();
+            //
+            //
+            //OddThreeDigit();
+            //
+            //
+            //XYZ();
+            //
+            //
+            //Calculator();
+            //
+            //
+            //HittingGap();
+            //
+            //
+            //Translate();
+            //
+            //
+            //Payment();
+            //
+            //
+            //CalculatorWithRepeat();
+            //
+            //
+            Factorial();
+            //
         }
 
-        public static decimal PerformCalculation(int r)
+        public static void HoursInDay()
         {
-            const decimal Pi = 3.14m;
-            decimal square = Pi * r * r;
-            return square;
-            
+            Console.WriteLine("Please enter a seconds");
+            string strseconds = Console.ReadLine();
+            int.TryParse(strseconds, out int seconds);
+            int hours = seconds / 3600;
+            Console.WriteLine($"Hours {hours}");
+        }
+
+        public static void ABetweenC()
+        {
+            Console.WriteLine("Enter A");
+            int a = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Enter B");
+            int b = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Enter C");
+            int c = Convert.ToInt32(Console.ReadLine());
+
+            if (a < b && b < c)
+            {
+                Console.WriteLine("True");
+            }
+            else
+            {
+                Console.WriteLine("False");
+            }
+        }
+
+        public static void OddThreeDigit()
+        {
+            string strthreedigit = Console.ReadLine();
+            int threedigit = Convert.ToInt32(strthreedigit);
+            if (threedigit % 2 == 0 && strthreedigit.Length == 3)
+            {
+                Console.WriteLine("True");
+            }
+            else
+            {
+                Console.WriteLine("False");
+            }
+        }
+
+        public static void XYZ()
+        {
+            int x = 10, y = 12, z = 3;
+            x += y - x++ * z;
+            Console.WriteLine(x);
+            z = --x - y * 5;
+            Console.WriteLine(z);
+            y /= x + 5 % z;
+            Console.WriteLine(y);
+            z = x++ + y * 5;
+            Console.WriteLine(z);
+            x = y - x++ * 2;
+            Console.WriteLine(x);
+        }
+
+        public static void Calculator()
+        {
+            Console.WriteLine("Enter first number");
+            int operand1 = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Enter second number");
+            int operand2 = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Enter arithmetic sign");
+            string sign = (Console.ReadLine());
```

Comment:
sign could contain more than 1 char. I thin we should add validation for that case.